main.js

```diff
@@ -1474,7 +1474,12 @@ addStealExtension(function (loader) {
 				});
 
 			} else {
-				loader["import"](loader.configMain)
+				var devBundlePath = (loader.devBundleDest || "") + "dev-bundle";
```

Comment:
@matthewp is there a helper already to join paths? so I don't have to add code to handle trailing slashes, and stuff like that?